installers/olm/README.md

```diff
@@ -1,6 +1,13 @@
 
-This directory contains the files needed to deploy (the Operator) using
-[Operator Lifecycle Manager][OLM].
+This directory contains the files that are used to install [Crunchy PostgreSQL for Kubernetes][hub-listing],
+which uses the PostgreSQL Operator, using [Operator Lifecycle Manager][OLM].
 
-[OLM]: https://github.com/operator-framework/operator-lifecycle-manager
+The integration centers around a [ClusterServiceVersion][olm-csv] [manifest](./postgresoperator.csv.yaml)
+that gets packaged for OperatorHub. Changes there are accepted only if they pass the all the [scorecard][]
```

Comment:
"~the~ all the"

Reply:
Fixed in amended https://github.com/CrunchyData/postgres-operator/pull/1259/commits/b824c3cd65e8637737413c1ed07df32d47da9117.